internal/venice-common/src/main/java/com/linkedin/venice/controllerapi/ControllerClient.java

```diff
@@ -837,6 +837,14 @@ public SchemaResponse getValueSchema(String storeName, int valueSchemaId) {
     return request(ControllerRoute.GET_VALUE_SCHEMA, params, SchemaResponse.class);
   }
 
+  public SchemaResponse updateValueSchema(String storeName, String valueSchema, int schemaId) {
+    QueryParams params = newParams().add(NAME, storeName)
+        .add(CLUSTER, clusterName)
```

Comment:
nit: this line could be removed, because cluster is already added in newParams() -> addCommonParams